src/App.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+import axios from 'axios'
+import View from './components/View'
+import Cards from './components/Cards'
+import { IData } from './models/dataModel.js'
+import './App.css'
+
+function App() {
+    const [newCustomerView, setnewCustomerView] = useState<boolean>(false)
+    const [data, setData] = useState<IData[] | undefined>()
+
+    const changeView = (): void => {
```

Comment:
Also here if we change only boolean value to opposite direction by naming convention it should be named as toggle*:
```tsx
const toggleCustomerView = (): void => {
    setIsNewCustomerView(!isNewCustomerView);
}
```